_data/fellows.yml

```diff
@@ -1,46 +1,53 @@
 - name: Will
   img: logo.jpg
-  location: London, UK
-
-- name: Amanda
-  img: amanda.jpg
-  location: New York City, USA
-
-- name: Aaron
-  img: aaron.jpg
-  location: London, UK
-
-- name: Swift
-  img: swift.jpg
-  location: New York City, USA
+  title: Will
+  url: http://google.fr
+  location: spain
+  coordinates: 
+    lat: 48.8587741
+    lng: 2.2074741
+    
+    
 
 - name: Arpit Bhalla
   img: mlh.jpg
   username: arpitBhalla
-  location: New York City, USA
   url: https://github.com/arpitbhalla
   social:
     twitter: arpit
     github: arpit
     linkedin: arpit
+    title: Aaron
```

Comment:
I think it would be better not to change this file. You're doing it dynamically so when the details are updated, it will automatically work here. So maybe you can change it back?

Reply:
Just added dummy coordinates and locations to the dummy data. Do you want me remove the dummy data?

Reply:
I just changed the dummy data to include dummy locations and coordinates. So do you want me to change it?

Reply:
Basically what I mean is that please don't change the fellows.yml file. Some people have already added their data, you can use that and display it

Reply:
Also please check the deploy logs, there is an error